packages/bson-transpilers/test/yaml/bson-methods.yaml

```diff
@@ -81,16 +89,19 @@ tests:
             shell: new ObjectId('5ab901c29ee65f5c8550c5b9').equals(new ObjectId('5ab901c29ee65f5c8550c5b9'))
             object: true
             ruby: BSON::ObjectId('5ab901c29ee65f5c8550c5b9') == BSON::ObjectId('5ab901c29ee65f5c8550c5b9')
+            swift: try BSONObjectID("5ab901c29ee65f5c8550c5b9") == try BSONObjectID("5ab901c29ee65f5c8550c5b9")
     -   input:
             python: ObjectId('5ab901c29ee65f5c8550c5b9') == ObjectId('5ab901c29ee65f5c8550c5b9')
         output:
             object: true
+            swift: try BSONObjectID("5ab901c29ee65f5c8550c5b9") == try BSONObjectID("5ab901c29ee65f5c8550c5b9")
```

Comment:
it's possible we get into trouble with `try`, since it might require the user to enclose whatever aggregation they have in a `throws` function. that said, having `try!` everywhere does seem to set a bit of a bad example though. not exactly sure what to do here, any thoughts?